DEVEL.md

```diff
@@ -122,6 +122,21 @@ Otherwise, they can be logged as simple values:
     klog.Warningf("Example of a warning due to an external error: %v", err)
 ```
 
+### Checklist for new device plugins
+
+For new device plugins contributed to this repository, below is a
+checklist to get the plugin on par feature and quality wise with
+others:
+
+1. Plugin binary available in [`cmd/`](cmd), its corresponding Dockerfile in [`build/docker/`](build/docker) and deployment Kustomization/YAMLs in [`deployments/`](deployments).
+2. Plugin binary Go unit tests implemented and passing with >80% coverage: `make test WHAT=./cmd/<device>`.
```

Comment:
Would it be better to use <plugin> instead of <device> here?

Reply:
changed to `<plugin>`